src/main/java/com/climatetree/places/controller/PlacesController.java

```diff
@@ -39,5 +39,10 @@ public String getSimilarPlaces(@PathVariable("placeId") int placeId,
 	public String getPlacesByName(@PathVariable("name") String name) {
 		return namesService.getPlacesBySearchTerm(name);
 	}
+
+	@GetMapping("/places/nearest")
+  public String getNearestPlace(@RequestParam double latitude, @RequestParam double longitude){
```

Comment:
Are these expected as URL parameters? if yes, then can we use `@PathVariable` instead?
Because that way, the URL pattern would be more intuitive.

Reply:
Yes parameters are required for the search. In the past I have used query parameters for searching and filtering, similar to how Google's search engine works, and path variables for identifying resources. Happy to make the change though. Would you suggest the URL be places/nearest/latitude/{latitude}/longitude/{longitude} ? Thanks!